marklogic-data-hub-central/ui/e2e/cypress/e2e/explore/sidebarIndicators.cy.tsx

```diff
@@ -9,25 +9,26 @@ describe("Test sidebar indicators", () => {
   before(() => {
     cy.visit("/");
     cy.contains(Application.title);
-
     cy.log("**Logging into the app as a developer**");
-    cy.loginAsDeveloper().withRequest().then(() => {
-      LoginPage.postLogin();
-    });
-  });
-  beforeEach(() => {
+    cy.loginAsDeveloper().withRequest();
+    LoginPage.postLogin();
```

Comment:
If we are visiting directly, we don'n need line 13

Reply:
We need 13 as it is setting the auth here but removed line 14